CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,54 @@
+
+- Please avoid statements about the presumed typical desires, capabilities or actions of some demographic group. They can offend people in that group, and they are always off-topic in USw-related discussions.
+
+- Please be especially kind to other contributors when saying they made a mistake. Programming means making lots of mistakes, and we all do so—this is why regression tests are useful. Conscientious programmers make mistakes, and then fix them. It is helpful to show contributors that being imperfect is normal, so we don't hold it against them, and that we appreciate their imperfect contributions though we hope they follow through by fixing any problems in them.
+
+- Please respond to what people actually said, not to exaggerations of their views. Your criticism will not be constructive if it is aimed at a target other than their real views.
+
+- If in a discussion someone brings up a tangent to the topic at hand, please keep the discussion on track by focusing on the current topic rather than the tangent. This is not to say that the tangent is bad, or not interesting to discuss—only that it shouldn't interfere with discussion of the issue at hand. In most cases, it is also off-topic, so those interested ought to discuss it somewhere else.
+
+- If you think the tangent is an important and pertinent issue, please bring it up as a separate discussion, with a Subject field to fit, and consider waiting for the end of the current discussion.
+
+- Rather than trying to have the last word, look for the times when there is no need to reply, perhaps because you already made the relevant point clear enough. If you know something about the game of Go, this analogy might clarify that: when the other player's move is not strong enough to require a direct response, it is advantageous to give it none and instead move elsewhere.
+
+- Please don't argue unceasingly for your preferred course of action when a decision for some other course has already been made. That tends to block the activity's progress.
+    
+- If others have irritated you, perhaps by disregarding these guidelines, please don't excoriate them, and especially please don't hold a grudge against them. The constructive approach is to encourage and help other people to do better. When they are trying to learn to do better, please give them plenty of chances.
+
+- If other participants complain about the way you express your ideas, please make an effort to cater to them. You can find ways to express the same points while making others more comfortable. You are more likely to persuade others if you don't arouse ire about secondary things.
+
+- Please don't raise unrelated political issues in USw-related discussions, because they are off-topic.
+
+- Don't be a dick.
```

Comment:
Why is this here?

Reply:
Request from @narcolepticinsomniac

Reply:
Was it an actual request or just a comment?

Reply:
I agree, good guideline. Consider it as an TL;DR

Reply:
I can't consider it as Tl;DR and neither can new contributors that are meant to read this document. As it stands, it doesn't belong there.

Reply:
> Was it an actual request or just a comment?

Offhanded remark. If anyone's taking this seriously, I promise you, I could not care less.

Reply:
Just as I thought. I'll let this PR stay open until tomorrow and that's when we'll decide what to do.